JS-grades/index.js

```diff
@@ -0,0 +1,12 @@
+const physicsGrade = 3;
+const mathGrade = 5;
+const englishGrade = 5;
+
+let gradeAverage = (physicsGrade + englishGrade + mathGrade)/3;
+
+console.log(gradeAverage);
+
+if (gradeAverage >= 4) {
+console.log('Good job!');
+} else if (gradeAverage < 4) {
+console.log('Try harder!');}
```

Comment:
When it comes to code formatting, make sure to use indentation, the same as with HTML or CSS :) 
```suggestion
if (gradeAverage >= 4) {
  console.log('Good job!');
} else if (gradeAverage < 4) {
  console.log('Try harder!');
}
```